src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/ConfigurationPath.cs

```diff
@@ -35,7 +36,7 @@ public static string Combine(params string[] pathSegments)
         /// </summary>
         /// <param name="pathSegments">The path segments to combine.</param>
         /// <returns>The combined path.</returns>
-        public static string Combine(IEnumerable<string> pathSegments)
+        public static string Combine([NotNull]IEnumerable<string> pathSegments)
```

Comment:
Why did you need to specify this? Isn't `pathSegments` not null by default?

Reply:
Thank you, I fixed it but not submitted in the Git Stage....